ENIGMAsystem/SHELL/Graphics_Systems/General/GStiles.cpp

```diff
@@ -91,6 +91,7 @@ namespace enigma
     {
         if (!tiles_are_dirty) return;
         tiles_are_dirty = false;
```

Comment:
If the tiles aren't dirty, why is the metadata gone?

Reply:
This is `load_tiles` which is called every `screen_redraw` and is supposed to return if the tiles are not dirty. If the tiles are dirty, the layer metadata (what `screen_redraw` uses to submit the index buffer), the vertex buffer, and index buffer all need to be cleared so we can load the new tiles.

Reply:
As a readability thing, I zero the dirty bit *after* load. But that's fine, I guess.